src/rules/memberAccessRule.ts

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-import { getChildOfKind, getModifier, getNextToken, getTokenAtPosition, isClassLikeDeclaration } from "tsutils";
+import * as utils from "tsutils";
```

Comment:
We usually tend to avoid namespace imports here. It's fine if the import list spans multiple lines.

Since this is only a matter of personal preference, I leave it up to you whether you want to change it or not.

Reply:
I found both ways in other files, I had to choose between multiple line import or namespace import, but I think I like multiline import more.